runtime/engine/src/main/moditect/module-info.java

```diff
@@ -23,6 +23,7 @@
     exports io.aklivity.zilla.runtime.engine.exporter;
     exports io.aklivity.zilla.runtime.engine.guard;
     exports io.aklivity.zilla.runtime.engine.metrics;
+    exports io.aklivity.zilla.runtime.engine.metrics.reader;
```

Comment:
My understanding is that metrics readers are moving to `metrics` command internal implementation (if still needed after readonly engine support) as they are not used by the engine or exporters.

Reply:
On the contrary, the `MetricsReader` class and the `*Record` classes in the `io.aklivity.zilla.runtime.engine.metrics.reader` package are being used by the metrics command, the prometheus exporter as well as the otlp exporter.

Reply:
Discussed to keep it as is for now.